src/librustdoc/html/static/styles/main.css

```diff
@@ -26,9 +26,6 @@ h1.fqn {
 h2, h3:not(.impl):not(.method):not(.type):not(.tymethod), h4:not(.method):not(.type):not(.tymethod) {
 	border-bottom-color: #DDDDDD;
 }
-.in-band {
-	background-color: white;
-}
```

Comment:
There is a reason for this behavior. When the text of a function overlaps a version number, the version number isn't visible anymore to avoid weird text overlap. Normally, you could just put your own `background-color` after this one without issues.